tests/push.bats

```diff
@@ -18,3 +18,24 @@ load helpers
     buildah rm "$cid"
   done
 }
+
+@test "push with manifest type" {
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  run buildah push --signature-policy ${TESTSDIR}/policy.json --format oci alpine dir:my-dir
+  echo "$output"
+  [ "$status" -eq 0 ]
+  manifest=$(cat my-dir/manifest.json)
+  run grep "application/vnd.oci.image.config.v1+json" <<< "$manifest"
```

Comment:
Is there a particular reason we're using redirection here, instead of giving `grep` the name of the file to read?

Reply:
No reason, I was just being silly. Fixed